### PR TITLE
Main fixes to 0.5.1 -> 0.5.1.post1

### PR DESCRIPTION
**Pull request by Gamenot**
This is a mini-release to fix the main problems with 0.5.1.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -67,13 +67,13 @@
...
             "pytest-xdist>=2.4.0",
+            "opencv-contrib-python-headless==4.1.2.30",
             "ray[rllib]==1.0.1.post1",  # We use Ray for our multiprocessing needs
             "tensorflow>=2.4.0",  # For rllib tests
         ],
         "train": [
-            "ray[rllib]==1.0.1.post1",  # We use Ray for our multiprocessing needs
-            # XXX: TF requires specific version of scipy
-            "scipy==1.4.1",
+            "opencv-contrib-python-headless==4.1.2.30",
```
Comment (RutvikGupta) [inline setup.py:75]: why was `scipy` removed?

Comment (RutvikGupta) [inline setup.py:75]: This seem to conflict with the changes in PR #1185.

image

**Review by RutvikGupta — APPROVED** [review]
Some minor concerns but looks good to go.

File: setup.py (hunk 1 above)
Comment (Gamenot) [reply setup.py:75]: `scipy` did not appear to be needed. I think I will return it for now.

Comment (Gamenot) [reply setup.py:75]: It will, I will merge master back into develop after this is merged in and undo these changes.

**Review by saulfield — APPROVED** [review]

**Pull request merged**